.gitignore

```diff
@@ -0,0 +1,6 @@
+workspace/build
+workspace/install
+workspace/log
+workspace/src/plugins
+.color_coded
+.ycm_extra_conf.py
```

Comment:
I think this two last files are files generated by your IDE, which are not relevant for the rest of the users.
To avoid having these in the common gitignore, you can create in the root directory of your computer a file called .gitignore_global that acts like a gitignore but in all projects you have in your PC. This also saves you time in including this type of files in every gitignore of each repo

Reply:
Cool!! I modify it :D